### ISSUE TITLE
Export in the wrong place ?

### ISSUE THREAD

**Issue by Cdk29** [henry090/fastai#114]
I am not 100 % sure it is a bug.

Running code from here : https://github.com/Cdk29/Cassava/blob/main/Xresnet.md

Particularly : 

I create the learner using : 

`learnR <- dataloader %>% cnn_learner(xresnet50(), metrics = accuracy(),  model_dir="Xresnet_files/") #prettier
`

But fastai export it elsewhere : 

`erolland@erolland-GL75-9SE:~/Bureau$ locate xresnet_50.pkl`

> /home/erolland/Bureau/Cassava/cassava-leaf-disease-classification/train_images/xresnet_50.pkl

I am not sure but I don't think it is the normal behavior of the export of fastai to use the directory of the training set and not the model_dir to export its model.

**Comment by henry090** [comment]
Could you do the same but with python and see that works as you intended? However, here is the path argument, could specify ur path here, as well? https://github.com/henry090/fastai/blob/a37c463c7ab9494729089924ac0f933439730651/R/image_loaders.R#L368-L373

**Comment by Cdk29** [comment]
Sorry I did not understood, my post was not maybe perfectly clear.

The export is made here : 

`learnR$export(fname = "xresnet_50.pkl")`

Since the learner is created like this : 

` model_dir="Xresnet_files/")`

I expected the exported model to go in the model directory.

**Comment by henry090** [comment]
Yes, I got it. I am not sure why. Could you run this code from the python side? If it works fine (same folder), then I can take a look
